docs/userguide/packaging/plug-ins.md

```diff
@@ -33,66 +33,74 @@ uninstall themselves before they are removed from memory and unloaded.
 The current plugin API is documented as
 [icuplug.h](http://icu-project.org/apiref/icu4c434/icuplug_8h.html)
```

Comment:
please fix apiref links like this -- (a) point to current stable api docs & (b) not version-specific :-)

Reply:
done